File: exemples/send-transaction/send_transaction.js
const xrpl = require("xrpl");
(async () => {
    const client = new xrpl.Client("wss://s.altnet.rippletest.net:51233");
    await client.connect();
    const wallet = xrpl.Wallet.fromSeed("s████████");
    const tx = {
        TransactionType: "Payment",
        Account: wallet.address,
        Destination: "r████████",
        Amount: xrpl.xrpToDrops("10")
    };
    const response = await client.submitAndWait(tx, { wallet });
    console.log("Result:", response.result.meta.TransactionResult);
    client.disconnect();
})();

File: exemples/send-transaction/send_transaction.py
from xrpl.clients import JsonRpcClient
from xrpl.models.transactions import Payment
from xrpl.transaction import submit_and_wait
from xrpl.wallet import Wallet

client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
wallet = Wallet(seed="s████████", sequence=0)
tx = Payment(account=wallet.classic_address, amount="1000000", destination="r████████")
response = submit_and_wait(tx, client, wallet)
print(response)
